source: in-house
Do not crash on "non-GNU system" (no GNU libc).

--- a/installer/dcheck.py	2015-08-26 04:52:23.900974542 -0700
+++ b/installer/dcheck.py	2015-08-26 04:56:35.111537299 -0700
@@ -365,9 +365,12 @@
         return '-'
     else:
 #        LIBC = ctypes.CDLL(ctypes.util.find_library('c'), use_errno=True)
-        LIBC = ctypes.CDLL(ctypes.util.find_library('c'),ctypes.DEFAULT_MODE,None, True)
-        LIBC.gnu_get_libc_version.restype = ctypes.c_char_p
-        return LIBC.gnu_get_libc_version()
+        try:
+            LIBC = ctypes.CDLL(ctypes.util.find_library('c'),ctypes.DEFAULT_MODE,None, True)
+            LIBC.gnu_get_libc_version.restype = ctypes.c_char_p
+            return LIBC.gnu_get_libc_version()
+        except:
+            return '-'
 
 def get_python_xml_version():
     try:
